ground/gcs/src/app/main.cpp

```diff
@@ -258,6 +272,18 @@ int main(int argc, char **argv)
 
     SharedTools::QtSingleApplication app((QLatin1String(appNameC)), argc, argv);
 
+#ifdef USE_CRASHREPORTING
+    QString dirName(GCS_REVISION_PRETTY);
+    dirName = dirName.replace("%@%", "_");
+
+    // Limit to boring characters
```

Comment:
When possible, it's good to specifically spell out what Regexp is looking for. It's a finicky language that many people are unfamiliar with.

Can the comment explain why we need to limit these characters?